test/quantization/core/test_quantized_op.py

```diff
@@ -1903,6 +1909,31 @@ def test_quantized_mean_qnnpack(self, keep):
             MQ = XQ.mean((2, 3), keepdim=keep)
             self.assertTrue(torch.equal(MQ, YQ))
 
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=2, max_dims=5,
```

Comment:
Although a lot of existing quantization tests use hypothesis for test case generation, this is now discouraged because the values hypothesis creates are not representative of real tensors.  Any chance we could switch the test to create the values without using hypothesis?  Here is an example: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L268

Unfortunately with the current setup, this test may not always catch any issues with the actual calculations for representative values inside the tensors.

Reply:
Thanks for the suggestion. I have reimplemented the test with `itertools.product`.